dojo/user/views.py

```diff
@@ -197,6 +199,7 @@ def alertcount(request):
 def view_profile(request):
     user = get_object_or_404(Dojo_User, pk=request.user.id)
     form = DojoUserForm(instance=user)
+    groups = get_groups_for_user(user)
```

Comment:
There is already another method you can use:
```suggestion
    groups = get_authorized_group_users_for_user(user)
```